ozaria/site/components/interactive/PageInteractive/common/ModalInteractive.vue

```diff
@@ -0,0 +1,90 @@
+<template>
```

Comment:
Is this specific to interactives modal, or a generic implementation of vue modals that can be used across all systems?

Reply:
It's a placeholder for the interactives specific format.  There may be parts of this that we can extract to a generic modal component.